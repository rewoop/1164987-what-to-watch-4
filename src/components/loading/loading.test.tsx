import React from "react";
import renderer from "react-test-renderer";
import Loading from "./loading.jsx";

it(`Should Loading render correctly`, () => {
  const tree = renderer
    .create(<Loading />).toJSON();

  expect(tree).toMatchSnapshot();
});
