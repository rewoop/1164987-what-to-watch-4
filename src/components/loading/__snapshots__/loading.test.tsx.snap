// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Loading render correctly 1`] = `
<div
  className="sweet-loading"
  style={
    Object {
      "backgroundColor": "#150202",
      "height": "100vh",
    }
  }
>
  <div
    className="css-1fnkusb"
  >
    <div
      className="css-qgh95i"
    />
    <div
      className="css-s09cay"
    />
    <div
      className="css-9nhjxe"
    />
    <div
      className="css-3emaeb"
    />
    <div
      className="css-nr88wv"
    />
    <div
      className="css-1m9fy0h"
    />
  </div>
</div>
`;
